.env

```diff
@@ -9,6 +9,26 @@ SENTRY_IMAGE=getsentry/sentry:nightly
 SNUBA_IMAGE=getsentry/snuba:nightly
 RELAY_IMAGE=getsentry/relay:nightly
 SYMBOLICATOR_IMAGE=getsentry/symbolicator:nightly
+
+# Sentry SaaS uses stock Yandex ClickHouse, but they don't provide images that
+# support ARM, which is relevant especially for Apple M1 laptops, Sentry's
+# standard developer environment. As a workaround, you can manually toggle the
+# following to get an ARM build from an alternative-but-reputable vendor.
+
+# See https://github.com/getsentry/self-hosted/issues/1385#issuecomment-1101824274
+
+CLICKHOUSE_IMAGE=yandex/clickhouse-server:20.3.9.70
+#CLICKHOUSE_IMAGE=altinity/clickhouse-server:20.10.1.4844-testing-arm
+# tl;dr ^^^ Uncomment this on ARM.
+
+# Images built on arm need to be tagged to use linux/arm64 on Apple silicon Macs.
+# To work around this issue we configure the platform here
+
+# See https://github.com/docker/cli/issues/3286 for the docker bug.
+DOCKER_PLATFORM=linux/amd64
```

Comment:
A potentially nicer solution than putting this and the clickhouse image in the `.env` would be to detect the docker architecture via `docker info --format '{{.Architecture}}'` and set them accordingly. I can add this if you think it is a good idea @chadwhitacre

Reply:
The less the manual work there is, the better this is so I'm a big 👍🏻

Reply:
@chadwhitacre https://github.com/getsentry/self-hosted/pull/1538/commits/8ef61db0a439c13c9bafdb30539174acb973da87 includes the changes I was working on for the auto-detection, I am not sure why the `turning-things-off.sh` script can't see the environment variables I set...